src/routing/authorisedRoute.component.test.tsx

```diff
@@ -35,15 +35,62 @@ describe('AuthorisedRoute component', () => {
     mockStore = configureStore();
   });
 
-  it('renders component when user logged in', () => {
+  it('renders non admin component when admin user accesses it', () => {
     state.scigateway.siteLoading = false;
     state.scigateway.authorisation.loading = false;
     state.scigateway.authorisation.provider = new TestAuthProvider(
       'test-token'
     );
 
     const AuthorisedComponent = withAuth(ComponentToProtect);
-    const wrapper = shallow(<AuthorisedComponent store={mockStore(state)} />);
+    const wrapper = shallow(
+      <AuthorisedComponent adminPlugin={false} store={mockStore(state)} />
+    );
+
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('renders non admin component when non admin user accesses it', () => {
```

Comment:
This test and the one above it have the same content. Is the `isAdmin` function supposed to be mocked in each to return a different boolean value from the other? How are the tests knowing the user is an admin or not? Or have I got my wires crossed?

Reply:
Good spot, thanks! Must've forgotten to change bits after pasting it.